src/DotNet.Testcontainers.Tests/Unit/Containers/Unix/Database/CouchbaseTestcontainerConfigurationTest.cs

```diff
@@ -1,93 +1,115 @@
 namespace DotNet.Testcontainers.Tests.Unit.Containers.Unix.Database
 {
   using System;
-  using Testcontainers.Containers.Configurations.Databases;
+  using DotNet.Testcontainers.Containers.Configurations.Databases;
   using Xunit;
 
   public class CouchbaseTestcontainerConfigurationTest
   {
     [Fact]
-    public void ShouldNotThrowsArgumentOutOfRangeException_WhenClusterConfigurationIsValid()
+    public void ShouldNotThrowArgumentOutOfRangeExceptionWhenClusterConfigurationIsValid()
     {
-      //Given
-      Action action = () => new CouchbaseTestcontainerConfiguration
+      // Given
+      CouchbaseTestcontainerConfiguration configuration = null;
+
+      Action action = () => configuration =  new CouchbaseTestcontainerConfiguration
       {
+        BucketName = "Bucket",
+        BucketType = "MEMCACHED",
```

Comment:
BucketName and BucketType has already default values and not required to set. These parameters is not validating. Also regarding Couchbase document, accepted bucket types are "couchbase", "ephemeral", and "memcached". The "couchbase" bucket is the standard bucket type. I'm not sure uppercase is valid.

Reply:
Thanks 👍. I added this to test the getters. We can improve this with further prs. Maybe with an enum or something like that.